assets/checks.py

```diff
@@ -1,12 +1,23 @@
-from discord import Member
+from discord import Member, Embed, Color
 from discord.ext.commands import Context, NoPrivateMessage
 from cogs.utils.constants import owner_ids
 
+import typing as t
+from cogs.utils import constants
+
 
 def owner(ctx: Context) -> bool:
     return ctx.author.id in owner_ids
 
 
+async def is_sudo(ctx: Context) -> t.Union[bool, None]:
+    if ctx.author.id in constants.devs:
+        return True
+    else:
+        embed = Embed(description="Make your Own sandwich Mortal!.", color=Color.red())
+        await ctx.send(embed=embed)
```

Comment:
```suggestion
    else:
        return True
```
don't react if the user is not a dev